llvm/lib/Target/Mips/MipsISelLowering.cpp

```diff
@@ -2875,6 +2878,15 @@ lowerGlobalTLSAddress(SDValue Op, SelectionDAG &DAG) const
   if (DAG.getTarget().useEmulatedTLS())
     return LowerToTLSEmulatedModel(GA, DAG);
 
+  // In CheriOS ABI we also use the captable for TLS
```

Comment:
Move down to use 
```
if (ABI.IsCheriOS()) {
} else  if (Subtarget.getABI().IsCheriPureCap()) {
}
```
Been a long time since I looked at this code so not quite sure if we should be using `Subtarget.getABI()` or `ABI`, but we should be consistent.